src/demo/level.rs

```diff
@@ -1,18 +1,22 @@
 //! Spawn the main level by triggering other observers.
```

Comment:
Outdated. Also, implementation details shouldn't be exposed in module-level docs.

Reply:
[`e56ec7d` (#242)](https://github.com/TheBevyFlock/bevy_quickstart/pull/242/commits/e56ec7d77cf71e25957602d5abe7b94b441c1895)